client/my-sites/email/controller.js

```diff
@@ -22,24 +20,22 @@ export default {
 	emailManagementAddGSuiteUsers( pageContext, next ) {
 		pageContext.primary = (
 			<CalypsoShoppingCartProvider>
-				<GSuiteAddUsers selectedDomainName={ pageContext.params.domain } />
+				<GSuiteAddUsers
+					productType={ pageContext.params.productType }
+					selectedDomainName={ pageContext.params.domain }
+				/>
 			</CalypsoShoppingCartProvider>
 		);
 
 		next();
 	},
 
-	emailManagementAddGSuiteUsersLegacyRedirect( pageContext ) {
-		page.redirect(
-			emailManagementAddGSuiteUsers( pageContext.params.site, pageContext.params.domain )
-		);
-	},
-
 	emailManagementNewGSuiteAccount( pageContext, next ) {
 		pageContext.primary = (
 			<CalypsoShoppingCartProvider>
 				<GSuiteAddUsers
-					planType={ pageContext.params.planType }
+					isNewAccount
+					productType={ pageContext.params.productType }
```

Comment:
Since we now always pass a product type to the `GSuiteAddUsers` component, we need a way to understand whether it's for a new account, or for extra licenses.